docs/cluster-management/configure-ui.md

```diff
@@ -112,4 +112,5 @@ If your Screwdriver Kubernetes Cluster is using [nginx Canary ingress](https://k
 | Environment name     | Default Value | Description          |
 |:---------------------|:--------------|:---------------------|
 | CANARY_RELEASE | "" | Set to "true" to denote  that this UI server is serving Canary version of UI |
-| RELEASE_VERSION | "stable" | UI Release version displayed under help menu in header|
\ No newline at end of file
+| RELEASE_VERSION | "stable" | UI Release version displayed under help menu in header|
+| SHOW_AVATARS | true | **Options:** (`true`/`false`) <br>Show user's avatar in build details page. |
```

Comment:
is this the right place for this variable?
seems like this section is more for canary routing